app/mapping/registration/TrusteeMapper.scala

```diff
@@ -45,7 +45,10 @@ class TrusteeMapper @Inject()(addressMapper: AddressMapper,
               name = indTrustee.name,
               dateOfBirth = indTrustee.dateOfBirth,
               phoneNumber = None,
-              identification = identificationMap(indTrustee)
+              identification = identificationMap(indTrustee),
+              nationality = indTrustee.nationality,
+              countryOfResidence = indTrustee.countryOfResidence,
+              legallyIncapableYesNo = indTrustee.legallyIncapableYesNo
```

Comment:
I believe this needs to be the inverse. It could be worth renaming your legally incapable yes no stuff to mental capacity yes no to make it clearer. If mental capacity is no then legally incapable is yes and vice versa

Reply:
Had already started working on that.